src/Relation/Binary/PropositionalEquality/Properties.agda

```diff
@@ -93,6 +93,9 @@ cong-∘ : ∀ {x y : A} {f : B → C} {g : A → B} (p : x ≡ y) →
          cong (f ∘ g) p ≡ cong f (cong g p)
 cong-∘ refl = refl
 
+sym-involutive : ∀ {x y : A} (p : x ≡ y) → sym (sym p) ≡ p
+sym-involutive refl = refl
```

Comment:
Amazed this wasn't already in the library!

Reply:
But have now made it local to `Data.List.Relation.Binary.Permutation.Propositional.Properties`, ditto.
```agda
trans-trans-sym : (p : x ≡ y) (q : y ≡ z) → ≡.trans (≡.trans p q) (≡.sym q) ≡ p
trans-trans-sym refl refl = refl
```